### PR TITLE
add method climatology

### PR DESCRIPTION
**Pull request by fabrivelas**
changed method anomaly to use the new method climatology
Please check

### CODE REVIEW

File: imktk/dataarray_methods/anomalies.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 import xarray as xr
+from imktk.dataarray_methods import climatology
 
 
 def main(dataarray):
     """Return monthly anomalies for dataarray."""
     assert isinstance(dataarray, xr.DataArray), "Input is not an xr.DataArray"
-    climatology = dataarray.groupby("time.month").mean("time")
-    anomalies = dataarray.groupby("time.month") - climatology
+    anomalies = dataarray.groupby("time.month") - climatology(dataarray)
```
Comment (ucyo) [inline imktk/dataarray_methods/anomalies.py:11]: Trying to execute this code returns the following error:

```sh
>>> t.imktk.anomalies()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/python/imktk/toolkit.py", line 54, in method
    return func(accessor._data, *args, **kwargs)
  File "/home/python/imktk/dataarray_methods/anomalies.py", line 11, in main
    anomalies = dataarray.groupby("time.month") - climatology(dataarray)
TypeError: 'module' object is not callable
```
Which reminds me of doing a proper testing environment: Related #38

**Review by ucyo — CHANGES_REQUESTED** [review]

File: imktk/dataarray_methods/anomalies.py (hunk 1 above)
Comment (ucyo) [reply imktk/dataarray_methods/anomalies.py:11]: Related #18

**Comment by ucyo** [comment]
Related #18

File: imktk/dataarray_methods/anomalies.py (hunk 1 above)
Comment (fabrivelas) [reply imktk/dataarray_methods/anomalies.py:11]: Did I do something wrong, here?

Comment (ucyo) [reply imktk/dataarray_methods/anomalies.py:11]: Yes. You did not use the function `main`. Try `from imktk.dataarray_methods.climatology import main as climatology`

**Comment by fabrivelas** [comment]
Hmm, I have to admit that I don’t understand :-)

**Comment by ucyo** [comment]
You are basically importing only the file and not the function

**Comment by fabrivelas** [comment]
Sorry for being ignorant, how do I import the function?

**Comment by ucyo** [comment]
There is a Turkish saying:

> Do not give me fish, teach me how to fish myself

**Comment by fabrivelas** [comment]
Ok, teach me :-)

**Comment by ucyo** [comment]
1. `main` is nothing different than a normal function. 
2. You can see the possible submodules and functions of an object `obj` by using `dir(obj)`

**Comment by ucyo** [comment]
Resolved with #85

**Pull request closed without merging**